Documentation/build-arm64-firmware.md

```diff
@@ -15,52 +15,52 @@ Note: The UEFI build environment has changed for 1903 and any existing build env
 
 1) Update and install build tools
     ```bash
-    $ sudo apt-get update
-    $ sudo apt-get upgrade
-    $ sudo apt-get install build-essential python python-dev python-crypto python-wand device-tree-compiler bison flex swig iasl uuid-dev wget git bc libssl-dev zlib1g-dev python3-pip
+    sudo apt-get update
+    sudo apt-get upgrade
+    sudo apt-get attr install build-essential python python-dev python-crypto python-wand device-tree-compiler bison flex swig iasl uuid-dev wget git bc libssl-dev zlib1g-dev python3-pip
     *** new for 1903 UEFI
-    $ sudo apt-get install gcc g++ make python3 mono-devel
+    sudo apt-get install gcc g++ make python3 mono-devel
     ***
-    $ pushd ~
-    $ wget https://releases.linaro.org/components/toolchain/binaries/7.2-2017.11/aarch64-linux-gnu/gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz
-    $ tar xf gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz
-    $ rm gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz
-    $ popd
+    pushd ~
```

Comment:
If you want to get rid of '~', then consider setting an environment variable with the current directory to use.  Ie:  export WORKSPACE=\`pwd\`

Reply:
I still like hosting the compiler in my home directory (but that's just me) so I would not totally get rid of ~ everywhere. If build script gets any more complicated I agree environment variable would be the next logical step.